src/hooks/spaces/useSpaceBySlug.ts

```diff
@@ -1,12 +1,17 @@
 import { useFirestore, useFirestoreCollectionData } from "reactfire";
-import Bugsnag from "@bugsnag/js";
 
-import { AnyVenue } from "types/venues";
+import { EMPTY_SPACE_SLUG, EMPTY_WORLD_SLUG } from "settings";
+
+import { World } from "api/world";
+
+import { AnyVenue, SpaceSlug } from "types/venues";
+import { WorldSlug } from "types/world";
 
 import { withIdConverter } from "utils/converters";
 import { WithId } from "utils/id";
 
 export type UseSpaceBySlugResult = {
+  world?: WithId<World>;
```

Comment:
Don't think this hook should return a world. That is an extra DB query in a hook to get a result that might not be even needed. Components can do with `useWorldBy` hooks if they need the world